src/pages/studyView/charts/StudyViewComponentLoader.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import {If} from 'react-if';
+import {ThreeBounce} from 'better-react-spinkit';
+import MobxPromise from "mobxpromise";
+
+export interface StudyViewComponentLoaderProps {
+    promise: MobxPromise<any>
+}
+
+export class StudyViewComponentLoader extends React.Component<StudyViewComponentLoaderProps> {
+    public render() {
+        return (
+            <div>
+                <If condition={this.props.promise.isPending}>
+                    <div style={{position: 'absolute', left: '50%', top: '40%'}}>
```

Comment:
we should do this centering with https://css-tricks.com/snippets/css/a-guide-to-flexbox/